src/test/java/pages/HomePage.java

```diff
@@ -1,20 +1,50 @@
 package pages;
 
 import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
 
 public class HomePage extends BasePage {
 
     By userAvatarIcon = By.cssSelector("img.avatar");
 
-    public HomePage( WebDriver givenDriver) {
+    public HomePage(WebDriver givenDriver) {
 
         super(givenDriver);
     }
 
-    public WebElement getUserAvatar () {
+    public WebElement getUserAvatar() {
         return findElement(userAvatarIcon);
     }
 
+    public void choosePlaylist() {
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".playlist:nth-child(3)")));
```

Comment:
We can make the xpath or css the class variable (Locator)

For example `private By playListLocator = By.cssSelector(".playlist:nth-child(3)");`